sematic/ui/src/components/SnackBarProvider.tsx

```diff
@@ -0,0 +1,90 @@
+import { Button, IconButton, Snackbar } from "@mui/material";
```

Comment:
This adds unified snack bar management across the app. Meaning any component can trigger a snack bar display (e.g. "Copied" notification, rerun trigger failure, new runs notifs, etc.)